docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
+- Create a new private app, with any “Private app name” and leaving the default permissions as read access under Admin API.
+- Enable the Etsy API by checking the box that says “Allow this app to access your storefront data using Storefront API”.
+- Make sure to also grant access to read product and customer tags by checking their corresponding boxes.
+
```

Comment:
You also need to enable two-factor authentication in order to create an API key.